src/logic/EligibilityTimelineCalculator.test.js

```diff
@@ -34,4 +34,27 @@ test.each(singleConvictionTestData)(
         expect(actualCalculatorOutput.getConviction(TEST_ID).vacatable).toBe(expectedVacatable);
     },);
 
+let duiConvictionTestData = [
+    ["Misdemeanor New Conviction Past 3 Years", CrimeClassification.MISDEMEANOR, false, false, "2018-01-01", "2019-01-01", "2019-11-01", false],
```

Comment:
This test case returns `false`, (i.e. not vacatable) when DUI is set to false. What do you think about adding another test case in which DUI is set to `true`?